scripts/enhance-requires

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+version_deps () {
+	while read dep
+	do
+		if [[ "${dep}" =~ ^[^\(]*\(\) ]] && rpm -q --whatprovides "${dep}" &> /dev/null
```

Comment:
`[[ =~ ]]` is a bash syntax, but the file is declared as pure-sh.

I think it's it's nicer to avoid a space after the redirection operator.

The conditional is not robust wrt. to failure. If `rpm -q` fails, we just take the other branch. If that is intentional, it'd be nice to add a comment explaining that.